Source/Source/Android/AdsBinding.java

```diff
@@ -11,6 +11,7 @@
 import com.scalemonk.ads.ScaleMonkAds;
 import com.scalemonk.ads.unity.banner.BannerContainerFactory;
 import com.scalemonk.libs.ads.core.domain.UserType;
+import com.scalemonk.libs.ads.core.domain.regulations.CoppaStatus;
```

Comment:
@marinoluck here, it's supposed to be done the same thing as GDPR, where there is an external enum class for the API and an internal for the SDK to use?